core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -536,7 +536,7 @@ private[log] class Cleaner(val id: Int,
     if(readBuffer.capacity >= maxIoBufferSize || writeBuffer.capacity >= maxIoBufferSize)
       throw new IllegalStateException("This log contains a message larger than maximum allowable size of %s.".format(maxIoBufferSize))
     val newSize = math.min(this.readBuffer.capacity * 2, maxIoBufferSize)
-    info("Growing cleaner I/O buffers from " + readBuffer.capacity + "bytes to " + newSize + " bytes.")
+    info("Growing cleaner I/O buffers from " + readBuffer.capacity + " bytes to " + newSize + " bytes.")
```

Comment:
Nit: Since you are changing this line, Scala string interpolation is preferred. 
Ex:
```scala
info(s"Growing cleaner I/O buffers from ${readBuffer.capacity} bytes to $newSize bytes.")
```